### ISSUE TITLE
Name property for list item of Combo-box in 'Create a new Workbook' window is not defined properly 

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/workbooks#342]
### Steps to Reproduce
1. Launch Xamarin Workbook application.
2. Navigate to 'Console' list item and select it
3. Navigate to Combo box and Expand it
4. Navigate through the list items and observe name property for list item

### Expected Behavior
Name property for list item of combo box should be properly defined (For e.g. .NET Core)

### Actual Behavior
Name property for list item is defined as 'Xamarin.Interactive.Client.ViewControllers.WorkbookAppviewController'  
**Narrator behavior:** Narrator is announcing list item as 'Xamarin.Interactive.Client.ViewControllers.WorkbookAppviewController, 2 of 2,not selected'
**NVDA behavior:** NVDA is announcing list item as 'Xamarin.Interactive.Client.ViewControllers.WorkbookAppviewController, 2 of 2,non selected'

### User Impact
Users who rely on screen reader will not be able to know the Name of list items and thus will not be able to select list item

### Screenshots

![list_item_name_property_inspect](https://user-images.githubusercontent.com/22956130/40722697-9daeb96c-643a-11e8-8a96-93aa40ef008b.png)
![list_item_name_property_narrator](https://user-images.githubusercontent.com/22956130/40722701-a1c1cdaa-643a-11e8-9a01-35cc025a6614.png)
![list_item_name_property_nvda](https://user-images.githubusercontent.com/22956130/40722707-a54b8cfe-643a-11e8-9bfd-88e6908abc74.png)

### Environment

Windows :Windows 10 Pro Version 1803 (OS Build : 17672.1000)
Xamarin Version : 1.4 (master/b3f92f9)
NVDA Version: 2018.1

**Comment by AccessibilityTestingTeam-TCS** [comment]
Exemption Approved - 1813
